cogboard-app/src/main/kotlin/com/cognifide/cogboard/storage/ContentRepository.kt

```diff
@@ -5,6 +5,12 @@ import java.io.File
 
 class ContentRepository(private val path: String = "/data/content") {
 
+    init {
+        if (!File(path).exists()) {
+            File(path).mkdir()
+        }
+    }
+
```

Comment:
One question. Why?

Reply:
because we changed initialization. Now, the files are copied from initData in case does not exists. But folder `content` is not created anywhere. As ContentRepository is responsible ONLY for content files. I dream about database not FILESYSTEM!. Maybe in future ;)

Reply:
Don't delete and develop! ;)